lib/components/modal/modal.less

```diff
@@ -1,11 +1,12 @@
 .s-modal {
-    --_mo-bg: fade(@black-legacy-900, 50%); // Background remains fixed
+    // TODO Replace this Less variable with a CSS variable or other solution
+    --_mo-bg: fade(.set-black()[600], 50%); // Background remains fixed
```

Comment:
Across the codebase, I've converted references to Less color variables that don't have equivalent CSS variables to reference values available in the color sets. Here, a static color value is needed in order to operate on the color.

Reply:
It looks like replacing fade with native css won't be exactly a walk in the park:
https://stackoverflow.com/questions/40010597/how-do-i-apply-opacity-to-a-css-color-variable

Reply:
Yep, for now I think our only reasonable course of action here is to reference the color directly in this `fade` function. Seeing as this one modifies a color value's opacity, I don't see a better short-term solution.

Long-term, we could use something like the proposed [relative color syntax](https://www.w3.org/TR/css-color-5/#relative-colors) to adjust opacity, but that won't be for a while.